### PR TITLE
S3C-885: Change v2 query auth window to 7 days

### PR DESCRIPTION
**Pull request by nicolas2bert**
https://github.com/scality/Arsenal/pull/370

### CODE REVIEW

**Comment by nicolas2bert** [comment]
@ironman-machine
CI_AGGRESSOR_FORCE_BUILD=1
try

**Comment by nicolas2bert** [comment]
@ironman-machine r+

**Comment by nicolas2bert** [comment]
@ironman-machine r+

**Comment by rahulreddy** [comment]
@ironman-machine r- r+

File: tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js [hunk 1]
```diff
@@ -39,10 +39,11 @@ describe('aws-node-sdk v2auth query tests', function testSuite() {
     });
 
     // AWS allows an expiry further in the future
-    // 100010 seconds is higher that the Expires time limit: 100000 seconds
+    // 604810 seconds is higher that the Expires time limit: 604800 seconds
```
Comment (dora-korpar) [inline tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js]: higher than*

**Comment by nicolas2bert** [comment]
@ironman-machine r- r+

**Comment by nicolas2bert** [comment]
Retried here: http://ci.ironmann.io/gh/scality/Integration/17196#build-parameters

**Pull request merged**